src/app/app.module.ts

```diff
@@ -12,10 +12,11 @@ import { AuthService } from './shared/auth-guard.service';
 import { UserService } from './shared/user.service';
 
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { MatCheckboxModule , MatButtonModule, MatInputModule, MatCardModule } from '@angular/material';
 
 @NgModule({
   imports: [
-    BrowserModule, AppRoutingModule, HttpClientModule, BrowserAnimationsModule
+    BrowserModule, AppRoutingModule, HttpClientModule, BrowserAnimationsModule, MatCheckboxModule, MatInputModule, MatCardModule

```

Comment:
These are already imported in the HomeModule as part of MaterialModule, so they should be removed from here.